index.js

```diff
@@ -128,6 +128,12 @@ module.exports = {
     "no-undef": "off",
     "no-unused-vars": "off",
 
+    // emotion
+    "emotion/jsx-import": "error",
```

Comment:
I'm not sure we want this rule, in my opinion the jsx pragma should be setup in babel and we should not have to import it every time manually. But first we would need to verify that setting it up in the babel config would not break the other components that don't use emotion's `css` function.